Gir_GstGL.toml

```diff
@@ -0,0 +1,385 @@
+[options]
+girs_dir = "gir-files"
+library = "GstGL"
+version = "1.0"
+min_cfg_version = "1.14"
+target_path = "gstreamer-gl"
+work_mode = "normal"
+concurrency = "send+sync"
+generate_safety_asserts = true
+single_version_file = true
+
+external_libraries = [
+    "GLib",
+    "GObject",
+    "Gst",
+    "GstBase",
+    "GstVideo",
+]
+
+generate = [
+    # BitFlags
+    "GstGL.GLAPI",
+    "GstGL.GLDisplayType",
+    "GstGL.GLPlatform",
+    "GstGL.GLSLProfile",
+    # Enums
+    "GstGL.GLFormat",
+    "GstGL.GLQueryType",
+    "GstGL.GLSLVersion", # https://github.com/gtk-rs/gir/issues/600
+    "GstGL.GLTextureTarget", # https://github.com/gtk-rs/gir/issues/600
+    "GstGL.GLStereoDownmix",
+    "GstGL.GLUploadReturn",
+    # Enums Errors
+    "GstGL.GLBaseMemoryError", # cannot find function gst_gl_base_buffer_error_quark()
+    "GstGL.GLContextError", # manually add error domain trait
+    "GstGL.GLSLError", # cannot find function gst_glsl_error_quark()
+    "GstGL.GLWindowError", # manually add error domain trait
```

Comment:
These 4 should be fine with the latest .gir file. Just be careful to *not* include the `get_type()` functions for all these as they will only be available from 1.16.

Reply:
done! :)

Reply:
Ok then the comments can be removed I guess? :)